content/blog/2019/08/2019-08-23-introduce-react-plugin-template.adoc

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: "Introduce React Plugin Template"
+tags:
+- react
+- plugins
+- template
+- gsoc
+- gsoc2019
+author: jackshen
+---
+The template's main repo is at 
+link:https://github.com/jenkinsci/react-plugin-template[React Plugin Template]
+
+This template is part of the project link:https://summerofcode.withgoogle.com/projects/#6112735123734528[Working Hours UI Improvement] during
+link:https://summerofcode.withgoogle.com/[Google Summer of Code 2019], which improve the UI of `Working Hours Plugin` using this pattern to develop Jenkins plugin with React. The `Working Hours Plugin` repository could be found at link:https://github.com/jenkinsci/working-hours-plugin[Working Hours Plugin].
+
+
+== Overview
+
+Developing plugin for Jenkins has always been an easy way to do with its Jelly based UI render system, but Jelly seems to be pretty heavy when we need to use some more modernized  tools like React and if we need to make the plugin UI much customized, this is what this template is built for.
+
+And with React integrated, development with Jenkins plugin is much modernized, developer could now use tons of React libraries, the way to use libraries is now tinier and safer with webpack, in short, coding with Jenkins plugin can be much easier.
+
+== Features
+
+[cols="2,8",options="header"]
+|=====================================
+| Feature               | Summary            
+| React Integrated      | React is integrated, your can take full control of the UI 
```

Comment:
Change `React is integrated, your can take full control of the UI` to `React is integrated, you can take full control of the UI`